partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/PlanTransform.kt

```diff
@@ -362,20 +403,31 @@ internal object PlanTransform {
 
             override fun visitRelOpExclude(node: Rel.Op.Exclude, ctx: Unit) = org.partiql.plan.Rel.Op.Exclude(
                 input = visitRel(node.input, ctx),
-                paths = node.paths.map { visitRelOpExcludePath(it, ctx) },
-            )
-
-            override fun visitRelOpExcludePath(node: Rel.Op.Exclude.Path, ctx: Unit): org.partiql.plan.Rel.Op.Exclude.Path {
-                val root = when (node.root) {
-                    is Rex.Op.Var.Unresolved -> org.partiql.plan.Rex.Op.Var(-1, -1) // unresolved in `PlanTyper` results in error
-                    is Rex.Op.Var.Local -> visitRexOpVarLocal(node.root, ctx)
-                    is Rex.Op.Var.Global -> error("EXCLUDE only disallows values coming from the input record.")
+                paths = node.paths.mapNotNull {
+                    val root = when (val root = it.root) {
+                        is Rex.Op.Var.Unresolved -> org.partiql.plan.Rex.Op.Var(-1, -1) // unresolved in `PlanTyper` results in error
```

Comment:
We should constructor an error and call visitRexOpErr. We don't want this to go unreported.

Reply:
I believe this is a dead branch. Given the the resolution happened in typing time. Going to throw an error.